comp/assign-mentors.yaml

```diff
@@ -0,0 +1,27 @@
+summary: Assign mentors to teams
+
+priority: should
+
+component: Competition
+
+milestone: $SRYYYY Competition
+
+area-owner: mentoring
+
+description: >-
+  We need to assign mentors to teams. This is a good way to ensure that
+  teams get the help they need.
+
+
+  We should assign mentors to teams as early as possible. This will
+  allow mentors to get to know their teams and help them prepare for
+  the competition.
+
+
+  Required actions:
+   - [ ] Inform teams that mentoring is available
+   - [ ] Create a list of teams that want mentoring
+   - [ ] Get mentors to sign up to mentor teams
+   - [ ] Assign mentors to teams
+   - [ ] Inform mentors and teams of the assignment
+   - [ ] Get feedback from mentors about team progress throughout the year
```

Comment:
This feels like it'd be better as a separate tasks, as there's a lot of time in between

Reply:
[38a62cc](https://github.com/srobo/recurring-tasks/pull/90/commits/38a62cc1f5ca07032cb3a3af9f7ce5c8ea4e02d3)